### ISSUE TITLE
Support for iOS 10

### ISSUE THREAD

**Issue by steipete** [kif-framework/KIF#876]
KIF currently does not work with iOS 10b1. Let's use this issue to track the status.

I haven't spent much time on it yet but the first problem here is that Apple removed the old accessibility inspector. These calls fail because the methods no longer exist:

```
self.initialAccessibilityInspectorSetting = [self.axSettingPrefController AXInspectorEnabled:nil]; [self.axSettingPrefController setAXInspectorEnabled:@(YES) specifier:nil];
```

There's no replacement because the whole inspector is gone - you can call `_shortMethodDecription` on the `AccessibilitySettingsController` object to see for yourself. The setting is also gone. The inspector is now a Mac app:

<img width="1126" alt="screen shot 2016-06-20 at 08 11 21" src="https://cloud.githubusercontent.com/assets/58493/16184770/46b5a01c-36bf-11e6-88cf-21adff88b9c9.png">

I'm not completely familiar with the codebase so we need to see how important the inspector is and if we can invoke the same touches without it. Did anyone already work on compatibility with Xcode 8/iOS 10b1?

**Comment by phatmann** [comment]
With #877 merged, do you still want to leave this PR open?

**Comment by steipete** [comment]
iOS 10 works now.

**Comment by phatmann** [comment]
Woot! Thanks for your help here.
